src/kernel-loader/index.ts

```diff
@@ -147,7 +147,7 @@ declare var globalThis: {
   ROLLOUTS?: Record<string, RolloutRecord>
 }
 
-globalThis.EXPLORER_BASE_URL = process.env.REACT_APP_EXPLORER_BASE_URL
+globalThis.EXPLORER_BASE_URL = (import.meta as any).env.VITE_APP_EXPLORER_BASE_URL
```

Comment:
Do we have a `.env.default` file where this environment variable, and other new environment variables that we're defining are set?